File: ui/src/app/model/MapDLVO.ts
import { Base } from "./Base";
import { Organisation } from "./Organisation";

export class MapDLVO extends Base {
    id: number;
    moduleId: number;
    moduleName: string;
    subModuleId: number;
    subModuleName: string;
    dlName: string;
    organisationId: number;

    constructor() {
        super();
        this.id = null;
        this.moduleId = null;
        this.subModuleId = null;
        this.dlName = '';
        this.moduleName = '';
        this.subModuleName = '';
        this.organisationId = null;
    }
}


export interface MapDLVOList {
    content: MapDLVO[];
    totalElements: number;
}

File: domain/src/main/java/com/centram/common/vo/MapDLVO.java
package com.centram.common.vo;


import com.centram.domain.DistributionList;
import lombok.*;

import java.io.Serializable;
import java.math.BigInteger;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
public class MapDLVO implements Serializable {
    private static final long serialVersionUID = -6554446568157662441L;
    private BigInteger id;
    private BigInteger moduleId;
    private String moduleName;
    private BigInteger subModuleId;
    private String subModuleName;
    private String dlName;
    private BigInteger organisationId;




}